app/uk/gov/hmrc/fhregistrationfrontend/views/summary/helpers/ContactPersonHelper.scala

```diff
@@ -79,14 +79,13 @@ object ContactPersonHelper {
         )
       )
 
-    val ContactPersonAddressLabel = {
-      val isUkAddress = bpr.businessAddress.country.contains("GB")
-      val useInternational = contactPersonForm.ukOtherAddress.contains(false)
-
-      if (useInternational || !isUkAddress)
+    val ContactPersonAddressLabel = (contactPersonForm.ukOtherAddress, bpr.businessAddress.country) match {
```

Comment:
Check on both
contactPersonForm.otherUkContactAddress,
contactPersonForm.otherInternationalContactAddress
for completeness - i think will be easier to read